### ISSUE TITLE
Plugin crushing in IDEA EAP 2016.3

### ISSUE THREAD

**Issue by gavocanov** [raniejade/spek-idea-plugin#12]
Hi!

When trying to run any test in IDEA EAP 2016.3 the plugin crashes with the following stack trace:

```
null
java.lang.IllegalAccessError
    at sun.misc.Unsafe.throwIllegalAccessError(Unsafe.java:1142)
    at com.intellij.execution.configurations.SearchScopeProvider.createSearchScope(SearchScopeProvider.java:32)
    at com.intellij.execution.configurations.CommandLineState.<init>(CommandLineState.java:55)
    at com.intellij.execution.configurations.JavaCommandLineState.<init>(JavaCommandLineState.java:28)
    at com.intellij.execution.application.BaseJavaApplicationCommandLineState.<init>(BaseJavaApplicationCommandLineState.java:39)
    at org.jetbrains.spek.idea.SpekRunConfiguration$getState$1.<init>(SpekRunConfiguration.kt:101)
    at org.jetbrains.spek.idea.SpekRunConfiguration.getState(SpekRunConfiguration.kt:101)
    at com.intellij.execution.runners.ExecutionEnvironment.getState(ExecutionEnvironment.java:158)
    at com.intellij.execution.runners.BaseProgramRunner.execute(BaseProgramRunner.java:55)
    at com.intellij.execution.runners.BaseProgramRunner.execute(BaseProgramRunner.java:50)
    at com.intellij.execution.ProgramRunnerUtil.executeConfiguration(ProgramRunnerUtil.java:118)
    at com.intellij.execution.impl.ExecutionManagerImpl.start(ExecutionManagerImpl.java:118)
    at com.intellij.execution.impl.ExecutionManagerImpl.access$300(ExecutionManagerImpl.java:68)
    at com.intellij.execution.impl.ExecutionManagerImpl$3.run(ExecutionManagerImpl.java:530)
    at com.intellij.util.concurrency.QueueProcessor.runSafely(QueueProcessor.java:223)
    at com.intellij.util.Alarm$Request$1.run(Alarm.java:387)
    at com.intellij.openapi.application.TransactionGuardImpl$6.run(TransactionGuardImpl.java:311)
    at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.runNextEvent(LaterInvocator.java:421)
    at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.run(LaterInvocator.java:405)
    at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
    at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:756)
    at java.awt.EventQueue.access$500(EventQueue.java:97)
    at java.awt.EventQueue$3.run(EventQueue.java:709)
    at java.awt.EventQueue$3.run(EventQueue.java:703)
    at java.security.AccessController.doPrivileged(Native Method)
    at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
    at java.awt.EventQueue.dispatchEvent(EventQueue.java:726)
    at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:838)
    at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:674)
    at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:386)
    at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
    at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
    at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
    at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
    at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
    at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
```

**Comment by raniejade** [comment]
Thanks, will get back to you when I replicate this.

**Comment by raniejade** [comment]
Should be fixed with #21
